static/app/views/dashboards/widgets/timeSeriesWidget/releaseBubbles/constants.tsx

```diff
@@ -1,10 +1,8 @@
 export const BUBBLE_SERIES_ID = '__release_bubble__';
 export const BUBBLE_AREA_SERIES_ID = '__release_bubble_area__';
-export const DEFAULT_BUBBLE_SIZE = 12;
+export const DEFAULT_BUBBLE_SIZE = 4;
```

Comment:
Previously, the bubble size was the entire height including whitespace around it. This was because I started w/ the chart configuration and calculated everything else based on that.

I inverted this so that we define the actual bubble pixel height and calculate the chart offsets based on that + padding.